solutions/swb-reference/integration-tests/tests/isolated/users/listUsers.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('list users negative tests', () => {
+  const setup: Setup = Setup.getSetup();
+  let adminSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('with invalid paginationToken', () => {
+    const pagToken = '1';
```

Comment:
`pag` is short for pagination where `page` is not short for pagination

neither is wrong though! since I am committing some stuff I will go ahead and change `pagToken` to `paginationToken` for simplicity's sake